suites/reef/rados/tier-2_rados_test-stretch-mode.yaml

```diff
@@ -1,231 +0,0 @@
-      module: test_cephadm.py
-      polarion-id: CEPH-83573746
-      config:
-        steps:
-          - config:
-              command: apply_spec
-              service: orch
-              validate-spec-services: true
-              specs:
-                - service_type: osd
-                  service_id: all-available-devices
-                  placement:
-                    label: osd
-                  spec:
-                    data_devices:
-                      all: "true"                         # boolean as string
-          - config:
-              command: shell
-              args: # display OSD tree
-                - "ceph osd tree"
-
-  - test:
-      name: MDS Service deployment with spec
-      desc: Add MDS services using spec file
-      module: test_cephadm.py
-      polarion-id: CEPH-83574728
-      config:
-        steps:
-          - config:
-              command: shell
-              args: # arguments to ceph orch
-                - ceph
-                - fs
-                - volume
-                - create
-                - cephfs
-          - config:
-              command: apply_spec
-              service: orch
-              validate-spec-services: true
-              specs:
-                - service_type: mds
-                  service_id: cephfs
-                  placement:
-                    label: mds
-
-  - test:
-      name: RGW Service deployment
-      desc: RGW Service deployment
-      module: test_cephadm.py
-      polarion-id: CEPH-83574728
-      config:
-        steps:
-          - config:
-              command: apply
-              service: rgw
-              pos_args:
-                - rgw.1
-              args:
-                placement:
-                  label: rgw
-
-  - test:
-      name: Configure client admin
-      desc: Configures client admin node on cluster
-      module: test_client.py
-      polarion-id:
-      config:
-        command: add
-        id: client.1                      # client Id (<type>.<Id>)
-        nodes:
-          - node8:
-              release: 7
-        install_packages:
-          - ceph-common
-          - ceph-base
-        copy_admin_keyring: true          # Copy admin keyring to node
-        caps: # authorize client capabilities
-          mon: "allow *"
-          osd: "allow *"
-          mds: "allow *"
-          mgr: "allow *"
-      abort-on-fail: true
-
-  - test:
-      name: Enable logging to file
-      module: rados_prep.py
-      config:
-        log_to_file: true
-      desc: Change config options to enable logging to file
-
-  - test:
-      name: Deploy stretch Cluster
-      module: test_stretch_deployment_with_placement.py
-      polarion-id: CEPH-83573621
-      config:
-        no_affinity: false
-        stretch_rule_name: stretch_rule
-        tiebreaker_mon_site_name: tiebreaker
-      desc: Enables connectivity mode and deploys cluster with Stretch rule with tiebreaker node
-      abort-on-fail: true
-
-  - test:
```

Comment:
All tests from **suites/reef/rados/tier-2_rados_test-stretch-mode.yaml** are removed from reef suites and not moved to other suite. Is this expected?

Reply:
`tier-2_rados_test-stretch-mode.yaml` was a subset of `tier-3_rados_test-location-stretch-mode.yaml` and therefore completely removed.
However, `Mon election strategies in stretch mode` test is not part of `tier-3_rados_test-location-stretch-mode.yaml`, very good catch